sklearn/ensemble/_iforest.py

```diff
@@ -107,10 +107,11 @@ class IsolationForest(OutlierMixin, BaseBagging):
            0.24.
 
     random_state : int, RandomState instance, default=None
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+        Determines the random number generation for selecting the feature
+        and the partition values that are used to generate each tree of
```

Comment:
I am not sure about the phrasing "partition values".

May Is suggest the following:

"Each tree in the isolation forest partitions the feature space by recursively selecting a feature and threshold uniformly at random for each node in the tree. random_state makes it possible to control the pseudo-randomness of that recursive process."